styles/pages/index.module.scss

```diff
@@ -57,13 +59,19 @@
 }
 
 .libraries {
+  --col-size: 25rem;
+
   align-items: center;
   display: grid;
-  grid-gap: 2rem 3rem;
-  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
-  justify-items: center;
+  grid-gap: 3rem;
+  grid-template-columns: repeat(auto-fit, minmax(var(--col-size), 1fr));
```

Comment:
Let's not use CSS variables here and be explicit here and in the media query. I get what you're doing here, but its just not a pattern we've done in this repo.